### ISSUE TITLE
`reportAbstractUsage` false negative when implementation is just a docstring

### ISSUE THREAD

**Issue by DetachHead** [DetachHead/basedpyright#1009]
Code sample in [basedpyright playground](https://basedpyright.com/?typeCheckingMode=all&code=GYJw9gtgBAhgRgYygSwgBzCALlAggIQGEAaWOAZyxBgSwgFMsALMAEwFgAoUSKLATzTIAdgHMU6TDjAA3eiBDJW9Lqs4IANjHLkoAMTBgAFAUIBKAFxcoNqAAF4larQbM2128uBRgho%2BXoNYDMoAFoAPigAOTBheitOWySoADo0jxsHCioaOkYWDkTPem84GBB-QOCwyJi4hOTbACIW7VZgFqaMqG6sp1zXAu6vKDKAL0qgkIjo2PjupOpkANmsAEl0DXoGYSx6VgBRBUw1TW1dfHKjAzBLXtl5RWVhkp8-AKma2fqF23IAVzQ8iMZhSvmMZnucgUShURRsIzKFQ%2B1RmdXm8OSAKBFVBSJBUAAxFBhGAoI8Tpi7A8Yc9MYiYBMUdNanMGo0oNjgXjGSCgA)

```python
from abc import ABC, abstractmethod
from typing import override


class Foo(ABC):
    @abstractmethod
    def foo(self) -> None:
        ...
    @abstractmethod
    def bar(self) -> None:
        """asdf"""
    
    @abstractmethod
    def baz(self) -> None:
        raise NotImplementedError

class Bar(Foo):
    @override
    def foo(self) -> None:
        super().foo()
    @override
    def bar(self) -> None:
        super().bar() # no error
    @override
    def baz(self) -> None:
        super().baz()
```

related: #630
